### PR TITLE
RAINCATCH-651 - Cancel Button does not cancel New Workflow operation

### PR DESCRIPTION
**Pull request by witmicko**
# Motivation
Bug in the template breaking cancel button.

# JIRA
[RAINCATCH-651](https://issues.jboss.org/browse/RAINCATCH-651)

### CODE REVIEW

File: dist/workflow-form.tpl.html.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ ngModule.run(['$templateCache', function ($templateCache) {
     '    <div class="md-toolbar-tools">\n' +
     '      <h3>{{ctrl.model.id ? \'Update\' : \'Create\'}} workflow</h3>\n' +
     '      <span flex></span>\n' +
-    '      <md-button class="md-icon-button" aria-label="Close" ng-click="ctrl.selectWorkflow($event, workflow)">\n' +
+    '      <md-button class="md-icon-button" aria-label="Close" ng-click="ctrl.selectWorkflow($event, ctrl.model)">\n' +
```
Comment (ghost) [inline dist/workflow-form.tpl.html.js:16]: think `selectWorkflow` could be even `selectItem` to be more generic if we want in future auto-generate angular modules

**Comment by vchepeli** [comment]
LGTM

**Review by vchepeli — COMMENTED** [review]
Reviewed

**Review by wtrocki — APPROVED** [review]
Reviewed. Verified on cluster.

**Pull request merged**